chainer/functions/noise/simplified_dropconnect.py

```diff
@@ -136,6 +142,9 @@ def simplified_dropconnect(x, W, b=None, ratio=.5, train=True, mask=None):
             Otherwise, The mask must be ``(n, M, N)`` shaped array.
```

Comment:
I think the same of `mask` should be `(M, N)` when `batchwise_mask` is `False`. Is that correct? If so, could you fix the description of `mask`?